doc/source/templates/04_finetuning_llms_with_deepspeed/create_dataset.py

```diff
@@ -0,0 +1,26 @@
+from datasets import load_dataset
+import json
+import os
+
+dataset = load_dataset("gsm8k", "main")
```

Comment:
yes. I don't want to add extra cognitive load to the data processing / ingestion part. This script is just using basic non-ray concepts to save the dataset so that the user understand that the next stage would just take the dataset in that expected jsonl format.